### ISSUE TITLE
Load theme file "default.min.css" only once and not for each SCEditor instance?

### ISSUE THREAD

**Issue by q2apro** [samclarke/SCEditor#739]
For each instance of SCEditor I am adding the options for the theme with `default.min.css`. 

So when there are 3 instances with:

```
sceditor.create(textarea_content, {
   plugins: 'dragdrop,plaintext,autosave,undo',
   dragdrop: dragdropOptions,
   format: 'xhtml',
   style: '../minified/themes/content/default.min.css',
   locale: 'de',
   icons: 'monocons',
  // ...
```

SCeditor loads 3 times the `default.min.css` file. 

How can we have it load only once (for performance)? (Actually i thought Chrome is caching the file load, but that does not seem to be the case.)

**Comment by brunoais** [comment]
I don't think that can be done because that style is loaded inside the iframe SCE is working in. Each editor instance has a different iframe for the WYSIWYG mode.
So.... I don't think such cache can be done without bringing hige compexity and potential problems related to CORS.

**Comment by q2apro** [comment]
In best case the browser will do the caching. But I see different requests in the developer console.

**Comment by brunoais** [comment]
Which browser? Both firefox and chrome show in the console even if using the cached versions

**Comment by q2apro** [comment]
Chrome. But in the developer console, by default there is "Disable cache" on. Turning it off it is visible that Chrome does cache the files.

But the requests remain (it seems). Loading is though 1 ms.
